client/service/src/client/wasm_override.rs

```diff
@@ -161,7 +161,13 @@ where
 				Some("wasm") => {
 					let wasm = WasmBlob::new(fs::read(&path).map_err(handle_err)?);
 					let version = Self::runtime_version(executor, &wasm, Some(128))?;
+					log::info!(target: "wasm_overrides",
+							   "Found wasm override in file: `{:?}`, version: {}",
+							   path.to_str(), version);
```

Comment:
```suggestion
					log::info!(
						target: "wasm_overrides",
						"Found wasm override in file: `{:?}`, version: {}",
						path.to_str(),
						version,
					);
```

Same goes for all your log macro calls.